app/konnect/analytics/use-cases.md

```diff
@@ -175,9 +171,8 @@ You learned that:
 * You can use information from reports to dig deeper into any anomalies, performance, or stability issues to determine what is causing the problem
 * After you create a report, you can export and share the reports with stakeholders to let them draw conclusions based on further analysis
 
-If you want to continue learning about ways to use {{site.konnect_short_name}} to analyze data, read the next use case [Diagnosing latency issues](/konnect/analytics/use-cases/latency/).
+If you want to continue learning about ways to use {{site.konnect_short_name}} to analyze data, read [Diagnosing latency issues](/konnect/analytics/use-cases/latency/).
 
 ## More information
```

Comment:
Should we add a link to the "Explorer" docs in here since it is the backbone for custom reports?